src/resources/cpp/language-basics.md

```diff
@@ -186,44 +194,51 @@ unsigned types, only supporting non-negative types.
-representation. The standard does not specify if `char` is signed or unsigned, but its signedness is determined by the
-compiler and is usually the most optimal representation for a single byte character on the platform. On x86-64, char is
-typically signed, whereas on ARM64, it is typically unsigned.
+representation, and are one byte large. The standard does not specify if `char` is signed or unsigned, but is is a
+distinct type. On x86-64, `char` is typically signed, whereas on ARM64, it is typically unsigned.
+
+Throughout the years, further character types were added to C++:
+
+- `wchar_t` - A character type that is typically wider than the `char` type. It is typically 32-bit on \*nix platforms,
+  and 16-bit on Windows.
+- `char8_t` - A character type that can represent any UTF-8 code unit.
+- `char16_t` - A character type that can represent any UTF-16 code unit.
+- `char32_t` - A character type that can represent any UTF-32 code unit.
 
-For characters that are not able to be represented in a single byte, C++ provides other distinct types for wider
-character storage.
+### Floating-Point Types
 
-- `wchar_t` - A character type that is typically wider than the `char` type. On \*nix platforms, it is typically 32 bits
-  in width. On windows platforms, it is typically 16 bits in width.
-- `char8_t` - A character type that can represent any UTF-8 character representation.
-- `char16_t` - A character type that can represent any UTF-16 code point.
-- `char32_t` - A character type that can represent any UTF-32 code point.
+A floating-point type can represent
 
-### Floating Point Types
+- integers (e.g. `0`),
+- numbers with fractional parts (e.g. `0.5`),
+- positive or negative infinity, and
+- NaN (not a number).
 
-A floating point type is a real number that can have a decimal part. However, floating point types cannot represent all
-decimal numbers, as computers are only capable of processing numbers of a fixed size. Thus, computers use various
-representations to approximate decimals.
+However, since floating-point numbers are fixed-size, they can only represent certain fractional numbers exactly. For
+example, `0.1` cannot be represented exactly by a floating-point number, and neither can be extremely tiny or extremely
+large numbers.
 
-There are three fundamental floating point types in C++, `float`, `double`, and `long double`. The range of
+There are at least three fundamental floating-point types in C++: `float`, `double`, and `long double`. The range of
 representable values for a `double` is required to be at least as large as that of a `float`, and the `long double` must
 have a range at least that of a `double`. The language does not provide many strong guarantees about the properties of
-the various floating point types.
+the various floating-point types.
 
-::: info IEEE-754 Floating Point Model
+::: info IEEE-754 floating-point Model
```

Comment:
Weird capitalization. The IEEE standard capitalizes both "F" and "P" with a hyphen between.

Reply:
Yeah, I think that's just a search/replace mistake from when I turned `Floating Point` into `floating-point`.